docs/UserGuide.md

```diff
@@ -102,3 +102,77 @@ Format: `deleteuser NAME`
 Examples:
 * `deleteuser Betsy` deletes the person with the username `Betsy`.
 * `deleteuser Betsy Hoover` deletes the person with the username `Betsy Hoover`.
+
+### Creating a group : `addgroup`
+
+Creates a new group with no members. 
+
+Format: `addgroup  GROUP` 
+
+* Creates a group with the specified GROUP
+
+Examples:
+
+Adds an existing contact to an existing group in TABS.
+
+* addgroup CS2103T
+
+### Adding a member : `addmember` 
+
+Format: `addmember GROUP NAME`
+
+Examples:
+
+* addmember CS2103T UserName
+* addmember CS2101 DefaultUser
```

Comment:
For subsequent examples, the commands should be in backticks i.e. follow format.